core/controllers/profile.py

```diff
@@ -480,6 +485,18 @@ def post(self):
 class SiteLanguageHandler(base.BaseHandler):
     """Changes the preferred system language in the user's preferences."""
 
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'PUT': {
+            'site_language_code': {
```

Comment:
Can we introduce regex validation for this param? This should be just two alphabetical characters right?

Reply:
Added a regex to check for pattern